web/crux/install-docker.sh.hbr

```diff
@@ -43,15 +45,15 @@ set_environment() {
       case $PLATFORM in
       'OSX')
         echo "Detected: Mac OS X"
-        PERSISTENCE_FOLDER=$HOME/Library/Dyrectorio/srv/dagent
+        PERSISTENCE_FOLDER=~/dyrectorio-files/srv/agent
         ;;
       'LINUX')
         echo "Detected: Linux"
-        PERSISTENCE_FOLDER=/srv/dagent
+        PERSISTENCE_FOLDER=~/dyrectorio-files/srv/dagent
```

Comment:
Nothing special, IMHO. But testers are welcome in Linux OS.

Reply:
~ resolves the home directory, which with root user it would resolve /root/dyrectorio-files/srv/dagent.

If you fine with it feel free to mark it resolved, but I'm not sure if it's a pretty solution. If we would store lot of data there I'd be against it, otherwise its a secret key, so it's makes some sense to place it in the root directory.

Reply:
I think `/root/dyrectorio-files/srv/dagent` better than `/srv/agent` (no information about the project / product) - but wait for others. cc. @nandor-magyar @m8vago what do u think?

Reply:
These are not dyrectorio files, but simply a somewhat organized persistence folder, `/srv/dagent` is not good, but this current one is not really an improvement either.

Reply:
Thanks for your TED Talk. `These are not dyrectorio files` is not true IMHO, these files are dyrectorio dependencies...

Reply:
Updated!